draw.typ

```diff
@@ -341,56 +344,164 @@
     },
     render: (ctx, start, end, ..ctrl) => {
       ctrl = ctrl.pos()
-      let f = if len == 1 {
-        t => util.bezier-quadratic-pt(start, end, ctrl.first(), t)
-      } else {
-        t => util.bezier-cubic-pt(start, end, ctrl.first(), ctrl.last(), t)
-      }
       cmd.path(
         ctx,
-        ..(
-          start,
-          ..range(1, samples).map(i => f(i/samples)),
-          end
-        ), 
+        (if len == 1 { "quad" } else { "cube" }, start, end, ..ctrl),
         fill: fill, stroke: stroke
       )
     }
   ),)
 }
 
-// Merge multiple paths
-#let merge-path(body, close: false, fill: auto, stroke: auto) = ((
+/// Create anchors along a path
+///
+/// - path (path): Path
+/// - anchors (positional): Dictionaries of the format:
+///     (name: string, pos: float)
+/// - name (string): Element name, uses paths name, if auto
+#let place-anchors(path, ..anchors, name: auto) = {
+  let name = name
+  if name == auto and "name" in path.first() {
+    name = path.first().name
+  }
+  ((
+    name: name,
+    children: path,
+    custom-anchors-drawables: (drawables) => {
+      if drawables.len() == 0 { return () }
+
+      let list = (:)
+      let s = drawables.first().segments
```

Comment:
There can. But you have to merge them to a single path first. This is because it does not really make sense to place anchors on different objects at different positions by using a percentage value along their paths.